### PR TITLE
[13.0] analytic_base_department: fixed and improved search views

### PR DESCRIPTION
**Pull request by mlaitinen**
- Fix searching analytic lines by department and account department
- Add "My Department" filters for both analytic accounts and analytic lines

### CODE REVIEW

File: analytic_base_department/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Base Analytic Department Categorization",
     "summary": "Add relationshet between Analytic and Department",
-    "version": "13.0.1.1.0",
+    "version": "13.0.1.2.0",
```
Comment (dreispt) [inline analytic_base_department/__manifest__.py:7]: FYI No need to manually bump version, this is done by the merge bot

**Review by dreispt — APPROVED** [review]

File: analytic_base_department/__manifest__.py (hunk 1 above)
Comment (mlaitinen) [reply analytic_base_department/__manifest__.py:7]: Good to know! I specifically went through the [guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers) to refresh my memory and I didn't see any mention about the merge bot.

Should I keep it as it is or undo this change?

**Review by marcelsavegnago — APPROVED** [review]

**Pull request merged**